datahub-frontend/app/auth/AuthUtils.java

```diff
@@ -112,4 +125,13 @@ public static Map<String, String> createSessionMap(final String userUrnStr, fina
 
     private AuthUtils() { }
 
+    private static Http.Cookie.SameSite convertSameSiteValue(@Nonnull final String sameSiteValue) {
+        try {
+            return Http.Cookie.SameSite.valueOf(sameSiteValue);
+        } catch (IllegalArgumentException e) {
+            log.warn(String.format("Invalid AUTH_COOKIE_SAME_SITE value: %s", sameSiteValue), e);
```

Comment:
Usually I like to include _what the code will do next_ inside warn / error logs like this:

Invalid SameSite value provided: %s. Using LAX instead.